src/praisonai-agents/praisonaiagents/plugins/manager.py

```
@@ -437,96 +443,221 @@ def get_single_file_plugin(self, name: str) -> Optional[Dict[str, Any]]:
+                    continue
+                for event, func in _adapt_plugin_hooks(plugin):
+                    registry.register_function(
+                        event=event,
+                        func=func,
+                        name=f"{name}:{event.value}",
+                    )
+                    count += 1
+                try:
+                    plugin._wired_into_registry = registry
+                except Exception:
+                    pass
```

Ruff (0.15.20)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)